tests/acceptance/setup/test.js

```diff
@@ -2,10 +2,10 @@ import {expect} from 'chai';
 import chai from 'chai';
 import Mugshot from 'mugshot';
 import chaiMugshot from 'chai-mugshot';
+import WebdriverIOAdapter from 'mugshot-webdriverio';
```

Comment:
I don't think that we need that. Correct me if I'm wrong: the reporter has `Mugshot` as devDep and `Mugshot` has `mugshot-webdriverio` as production dep. By default `mugshot-webdriverio` is installed on `npm i`.

That error it is suppose to be like that:
https://github.com/uberVU/mocha-mugshot-reporter/blob/47-upgrade-mugshot/tests/acceptance/setup/test.js#L67
I wanted to test that even if the baseline/diffs are not generated (aka I forgot to give mugshot a browser), the error message will still be displayed in the reporter.

I will create an issue to develop support for generating diffs when the assertion is negated but we still have to figure out how we gonna test something like above or if we can ditch that.

#48 is about deleting the visual-report folder created by the tests so it starts fresh every time.